docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -28,9 +35,20 @@ There are three basic tasks to converting an add-in project from the add-in only
 
 The following sections describe conditions that must be met before you convert the manifest.
 
-### Ensure that you have the two image files
+### Uninstall the existing version of the add-in
+
+To avoid conflicts with UI control names and other problems, be sure the existing add-in isn't installed on the computer where you do the conversion.
+
+### Ensure that you have two special image files
+
+If your add-in only manifest doesn't already have both **\<IconUrl\>** and **\<HighResolutionIconUrl\>** (in that order) elements, then add them just below the **\<Description\>** element. The values of the **DefaultValue** attribute should be the full URLs of image files. The images must be a specified size as shown in the following table. 
 
-When you've added the files to the project, add **\<IconUrl\>** and **\<HighResolutionIconUrl\>** (in that order) to the add-in only manifest just below the **\<Description\>** element. The following is an example.
+|Office application|**\<IconUrl\>**|**\<HighResolutionIconUrl\>**|
```

Comment:
You shouldn't need to bold these since Learn automatically bolds the header
 
```suggestion
|Office application|\<IconUrl\>|\<HighResolutionIconUrl\>|
```